hoomd/hpmc/IntegratorHPMCMono.h

```diff
@@ -948,7 +948,9 @@ void IntegratorHPMCMono<Shape>::update(uint64_t timestep)
                     this->computeOneExternalEnergy(typ_i, pos_i, shape_i.orientation, h_charge.data[i], true);
                 }
 
-            bool accept = !overlap && hoomd::detail::generate_canonical<double>(rng_i) < slow::exp(patch_field_energy_diff);
+            Scalar kT = IntegratorHPMC::getTimestepkT(timestep);
```

Comment:
```suggestion
            const Scalar kT = (*m_mc->getKT())(timestep);
```
Also, move this line to the top of the function. `kT` does not vary during the loop, so there is no need to evaluate it every loop iteration.